src/main/java/com/sap/sgs/phosphor/fosstars/model/tuning/CMAESWeightsOptimization.java

```diff
@@ -293,7 +300,7 @@ public double value(double[] parameters) {
     }
 
     /**
-     * @return A number of weights which may be adjusted.
+     * Returns a number of weights which may be adjusted.
```

Comment:
Returns *the number of weights* maybe? as we are specific about weights here?  :)

Reply:
I agree.